index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
This is title of the game
```suggestion
This is a paragraph
```